App/Sources/Feature/StudentSignUpFeature/StudentSignUpViewModel.swift

```diff
@@ -0,0 +1,192 @@
+            return "재학 중이신 학교를 선택해 주세요!"
+        } else if selectedClub == "동아리" {
+            return "가입하신 동아리를 선택해 주세요!"
+        } else if name.isEmpty {
+            return "이름을 입력해 주세요!"
+        } else if yearOfAdmission.isEmpty {
+            return "입학하신 연도를 입력해 주세요!"
+        } else if studentID.isEmpty {
+            return "학년, 반, 번호를 입력해 주세요! (ex: 1101)"
+        } else if phoneNumber.isEmpty {
+            return "인증을 위해 전화번호를 입력해 주세요!"
+        } else if certificationNumberPhoneNumber.isEmpty {
+            return "받으신 인증번호 N자리를 입력해 주세요!"
+        } else if email.isEmpty {
+            return "이메일을 입력해 주세요!"
+        } else if certificationNumberEmail.isEmpty {
+            return "받으신 인증번호 N자리를 입력해 주세요!"
+        } else if password.isEmpty {
+            return "8~24자 이내의 영문 / 숫자, 특수문자 1개 이상"
+        } else {
+            return "비밀번호를 다시 입력해 주세요!"
+        }
+    }
+    
+    var selectedSchoolExists: Bool {
+        selectedSchool != "학교"
+    }
+    
+    var selectedClubExists: Bool {
+        selectedClub != "동아리"
+    }
+    
+    func convertSecondsToTime(timeInSeconds: Int) -> String {
+        let minutes = timeInSeconds / 60
+        let seconds = timeInSeconds % 60
+        return String(format: "%02i:%02i", minutes, seconds)
+    }
+    
+    func phoneNumberStartTimer() {
+        guard timer == nil else { return }
+        
+        timer = Timer.scheduledTimer(withTimeInterval: 1.0, repeats: true) { _ in
+            DispatchQueue.main.async {
+                if self.phoneNumberTimeRemaining > 0 {
+                    self.phoneNumberTimeRemaining -= 1
+                } else {
+                    self.stopTimer()
+                }
+            }
+        }
+    }
+    
+    func emailStartTimer() {
+        guard timer == nil else { return }
+        
+        timer = Timer.scheduledTimer(withTimeInterval: 1.0, repeats: true) { _ in
+            DispatchQueue.main.async {
+                if self.emailTimeRemaining > 0 {
+                    self.emailTimeRemaining -= 1
+                } else {
+                    self.stopTimer()
+                }
+            }
+        }
+    }
+    
+    func stopTimer() {
+        timer?.invalidate()
+        timer = nil
+    }
+    
+    public func parseStudentID() {
+        guard studentID.count == 4, 
+              let grade = Int(String(studentID.prefix(1))),
+              let classNumber = Int(String(studentID.dropFirst(1).prefix(1))),
+              let studentNumber = Int(String(studentID.suffix(2))) 
+        else { return }
+        
+        studentID = "\(grade)학년 \(classNumber)반 \(studentNumber)번"
+    }
+    
+    public func parsePhoneNumber() {
+        guard phoneNumber.count == 11, let firstPart = Int(String(phoneNumber.prefix(3))), let secondPart = Int(String(phoneNumber.dropFirst(3).prefix(4))), let thirdPart = Int(String(phoneNumber.dropFirst(7))) else { return }
+        
+        phoneNumber = "\(firstPart)-\(secondPart)-\(thirdPart)"
+    }
```

Comment:
```suggestion
        guard 
        phoneNumber.count == 11, 
        let firstPart = Int(String(phoneNumber.prefix(3))), 
        let secondPart = Int(String(phoneNumber.dropFirst(3).prefix(4))), 
        let thirdPart = Int(String(phoneNumber.dropFirst(7))) else { return }
        
        phoneNumber = "\(firstPart)-\(secondPart)-\(thirdPart)"
    }
```
이런 식으로 더 읽기 좋게 바꿀 수 없을까요??